packages/loader/src/resource-deserialize/resources/texture2D/Texture2DDecoder.ts

```
@@ -19,57 +20,32 @@ export class Texture2DDecoder {
-      const mipCount = bufferReader.nextUint8();
-      const imagesData = bufferReader.nextImagesData(mipCount);
+      const imageData = bufferReader.nextImagesData(1)[0];
 
-      const texture2D = restoredTexture || new Texture2D(engine, width, height, format, mipmap, isSRGBColorSpace);
-      texture2D.filterMode = filterMode;
-      texture2D.anisoLevel = anisoLevel;
-      texture2D.wrapModeU = wrapModeU;
-      texture2D.wrapModeV = wrapModeV;
+      const isHDR = imageData[0] === 0x23 && imageData[1] === 0x3f;
+      const textureFormat = isHDR ? TextureFormat.R16G16B16A16 : format;
+      const texture = restoredTexture || new Texture2D(engine, width, height, textureFormat, mipmap, isHDR ? false : isSRGBColorSpace);
```

ESLint

[error] Replace `·restoredTexture·||` with `⏎········restoredTexture·||⏎·······`

(prettier/prettier)